www/js/diary/LabelTab.tsx

```diff
@@ -70,19 +71,33 @@ const LabelTab = () => {
       setFilterInputs(allFalseFilters);
     }
     loadTimelineEntries();
-  }, [appConfig, loading, refreshTime]);
+  }, [appConfig, refreshTime]);
```

Comment:
I don't think it really matters either way.
`loading` is a boolean for whether the `appConfig` is still being loaded. Any time the value of `loading` changes, the value of `appConfig` also changes, so it's redundant to watch both.

Reply:
I think I see.

My concern was that since 

```
const [isLoading, setIsLoading] = useState<string|false>('replace');
```

loading would initially be set to `replace`

and so we would always hit the early return in the effect

```
  useEffect(() => {
     if (loading) return;
  }, [appConfig, refreshTime]);
```

I do see a `setIsLoading(false)` but that is when the entries are loaded, not the appconfig

```
  useEffect(() => {
    if (!displayedEntries?.length) return;
    invalidateMaps();
    setIsLoading(false);
  }, [displayedEntries]);
```

Can't there be a case in which the appconfig updates all happen before the trip load is complete?
Then when the appconfig use effect is called, it would early return
But when the displayedEntries is refreshed, the `appConfig` useEffect would never be called (since it is not a dependency)

even if this works now, you might want to refactor in a future change to split app loading from trip loading for ease of maintenance.

Reply:
I should definitely rename these variables in a later refactoring `isLoading` and `loading` are two different things.

`isLoading` keeps track of state so the loading spinner knows where/how to appear, while `loading` is just for the `appConfig`